packages/permission-controller/src/rpc-methods/getPermissions.ts

```diff
@@ -8,7 +8,7 @@ import { MethodNames } from '../utils';
 
 export const getPermissionsHandler: PermittedHandlerExport<
   GetPermissionsHooks,
-  undefined,
+  any,
```

Comment:
```suggestion
[],
```

Using `[]` as the empty value here also resolves the compiler errors. It's a little hacky, but I think it should be safer than using `any`.